.github/workflows/ci.yml

```diff
@@ -26,9 +26,11 @@ jobs:
           - 2.5
           - 2.4
           - 2.3
+          - jruby-9.2
```

Comment:
This worked fine https://github.com/rspec/rspec-support/pull/436/checks?check_run_id=1418962579

Are you interested in incorporating this in your GHA initiative?

I've removed `JRUBY_OPTS='--dev'` with no replacement for GHA. Not sure if it's needed.

The reason I'm doing this in this PR (temporarily) is to test this out while Travis is 🐌 
```
      # To work around JRuby error:
      # can't convert RSpec::Support::StdErrSplitter into String
```

Reply:
Sorry, forgot to mention @JonRowe ☝️

Reply:
`jruby-head` failure seems to be due:

```
Warning: the --server flag is deprecated and has no effect most JVMs
```

[This will be fixed in 9.3.0.0](https://github.com/jruby/jruby/issues/6147)

Wondering why it doesn't affect our `jruby-9.2` 😕

Reply:
And https://github.com/rspec/rspec-support/commit/33e6318 didn't help, since it outputs to stdout 😞

Reply:
I'd leave this the same as main but remove the 2 old rubies.